content/doc/help/troubleshooting.en-us.md

```diff
@@ -0,0 +1,69 @@
+---
+date: "2016-11-08T16:00:00+02:00"
+title: "Troubleshooting"
+slug: "troubleshooting"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "Help"
+    name: "Troubleshooting"
+    weight: 20
+    identifier: "troubleshooting"
+---
+
+# Troubleshooting
+
+This page contains some common issues you can run into and their solutions.
+
+## SSH issues
+
+If you are having issues with reaching your repositories over `ssh` while the
+Gitea web front-end and `https` based git operations work fine, consider
+looking at the following items.
+
+```
+Permission denied (publickey).
+fatal: Could not read from remote repository.
+
+Please make sure you have the correct access rights
+and the repository exists.
+```
+
+This error signifies that the server rejected your log in attempt, check the
+following things:
+
+* On the client:
+  * Ensure the public and private ssh keys are added to the correct Gitea user.
+  * Make sure there are no issues in your remote url, ensure the name of the
+    git user (before the `@`) is spelled correctly.
+  * Ensure the public and private ssh keys are available and reachable on the
+    client machine.
+  * Try to `ssh git@myremote.example` to ensure that everything is set up
+    properly.
+* On the server:
+  * Check the permissions of the `.ssh` directory in the home directory of your
+    `git` user.
+  * Verify that the correct public keys are added to `.ssh/authorized_keys`.
+    Try to run `Rewrite '.ssh/authorized_keys' file (for Gitea SSH keys)` on the
+    Gitea admin panel.
+
+If you get a similar error, but without the public key part (shown below),
```

Comment:
there should be no comma before `but`. Also two 'but' in same sentence is not good style and makes it hard to understand. Probably removing comma after `(shown below)` and adding `then` in it's place would be better

Reply:
Good points; I made some minor changes to the text based on your feedback.